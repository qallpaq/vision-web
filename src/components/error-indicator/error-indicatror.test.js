import React from 'react'
import ErrorIndicator from "./error-indicator"
import {shallow} from "enzyme"


describe('ErrorIndicator', () => {
    const wrapper = shallow(<ErrorIndicator/>)

    it('should return snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    })

    it('should contain .error-page__title', () => {
        const result = wrapper.find('.error-page__title')
        expect(result.text()).toBe('Something goes wrong...')
    })

    it('should contain .error-page__subtitle', () => {
        const result = wrapper.find('.error-page__subtitle')
        expect(result.text()).toBe('don`t worry we will fix it')
    })
})
