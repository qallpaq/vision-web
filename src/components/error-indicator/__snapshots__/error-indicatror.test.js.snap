// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorIndicator should return snapshot 1`] = `
<div
  className="error-page"
>
  <div
    className="error-page__content"
  >
    <h1
      className="error-page__title"
    >
      Something goes wrong...
    </h1>
    <div
      className="error-page__subtitle"
    >
      don\`t worry we will fix it
    </div>
  </div>
</div>
`;

exports[`ads 1`] = `
<div
  className="error-page"
>
  <div
    className="error-page__content"
  >
    <h1
      className="error-page__title"
    >
      Something goes wrong...
    </h1>
    <div
      className="error-page__subtitle"
    >
      don\`t worry we will fix it
    </div>
  </div>
</div>
`;
